Gradle.md

```diff
@@ -8,9 +8,47 @@ There are currently five experiments for Gradle. You could also perform these ex
 
 ## Requirements
 
-You need to have [Bash](https://www.gnu.org/software/bash/) installed in order to run the build validation scripts.
+You must have the following tooling in order to use the Develocity Build Validation Scripts:
 
-If you plan to use the build validation scripts on Windows, then you will need to [install Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/install). The build validation scripts work well on the WSL default distribution (Ubuntu). Note how to deal with the eventuality of facing filename too long errors [here](#dealing-with-filename-too-long-errors-on-windows).
+- [Develocity](https://gradle.com/develocity/)
+- [Bash](https://www.gnu.org/software/bash/)
+
+For users running on Windows, you will need to [install Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/install).
+The scripts work well on the WSL default distribution (Ubuntu).
+Note how to deal with the eventuality of facing filename too long errors [here](#dealing-with-filename-too-long-errors-on-windows).
+
+### User permissions
+
+Experiment builds running locally require different permissions than those that run in CI.
+Note that while not all permissions are required, they're still strongly recommended for the best possible experience.
+
+Your Develocity user should be granted the following permissions:
+
+| User permission                 | Reason                                                                                                                                      | Required              |
+|---------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------|-----------------------|
+| Publish build scans             | Analyzing the experiment results relies on build scans published to Develocity.                                                             | Yes                   |
+| Read build cache data           | Reading from the remote build cache.                                                                                                        | Only for experiment 5 |
```

Comment:
Exp 4 (ci/ci), too. No?

Reply:
I assume you are referring to running locally only.

Reply:
Ha, it is clarified further down. Maybe split local vs. CI into subtitles?

Reply:
> Maybe split local vs. CI into subtitles?

Can you clarify what you mean?

Reply:
Under `### User permissions`, have `#### Running the scripts locally` and `#### Running the scripts on CI`.